inode/locker_test.go

```diff
@@ -1,27 +1,32 @@
-package fs
+package inode
 
 import (
 	"testing"
 
 	"github.com/swiftstack/ProxyFS/blunder"
 	"github.com/swiftstack/ProxyFS/dlm"
-	"github.com/swiftstack/ProxyFS/inode"
 )
 
 func TestLockerStuff(t *testing.T) {
 
-	var inodeNumber inode.InodeNumber = inode.InodeNumber(2)
+	var inodeNumber = InodeNumber(2)
+	testVolumeHandle, err := FetchVolumeHandle("TestVolume")
+	if nil != err {
+		t.Fatalf("FetchVolumeHandle(\"TestVolume\") failed: %v", err)
+	}
+
+	volume := testVolumeHandle.(*volumeStruct)
 
 	// Test: get a lock for vol A inode 2
-	srcDirLock, err := mS.volStruct.initInodeLock(inodeNumber, nil)
+	srcDirLock, err := InitInodeLock(volume.volumeName, inodeNumber, nil)
```

Comment:
Yeah, here is the first example... I think fs's volume should already have an inode VolumeHandle, right?